SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphIngestionProcessor.cs

```diff
@@ -285,7 +309,7 @@ private async Task GetPointsAsync(IDictionary<string, BasicDigitalTwin> twins, I
                     switch (innerElement.Value.ValueKind)
                     {
                         case JsonValueKind.Array:
-                            foreach (var item in innerElement.Value.EnumerateArray())
+                            foreach (var item in innerElement.Value.EnumerateArray().Where(x => x.ValueKind != JsonValueKind.String))
```

Comment:
What happens to the string values?

Reply:
This call tells the app to traverse another level of the graph... since there isn't one if this is a string, we skip it since we don't want to navigate down another level. Will verify that the actual values are being stored elsewhere.

Reply:
This seems to be an issue with the vagueness of GetPlacesAsync taking in a JsonElement for the `targetElement` parameter rather than a well defined schema. To avoid the case of taking in a Number, maybe the where clause should be `.Where(x => x.ValueKind == JsonValueKind.Object)` since the first thing this method does is check for an `id` which a JsonValueKind.Number would not have.